spec/features/shelters/edit_spec.rb

```diff
@@ -20,13 +23,13 @@
   end
 
   it "has a flash message if a form is edited incompletely" do
-    fill_in :name, with: ""
-    fill_in :address, with: "456 Dublin St"
+    fill_in :name, with: "dd"
+    fill_in :address, with: ""
     fill_in :city, with: "Phoenix"
     fill_in :state, with: ""
     fill_in :zip, with: "60341"
     click_button "Update #{@shelter_1.name}"
     expect(current_path).to eq("/shelters/#{@shelter_1.id}/edit")
-    expect(page).to have_content("The following fields are missing : name, state")
```

Comment:
testing for the standard error sentence produced by the call `error.full_message.to_sentence`